lua/jupynium/init.lua

```diff
@@ -67,6 +66,20 @@ function M.setup(opts)
     server.register_autostart_autocmds(augroup, options.opts)
   end
 
+  local filename = vim.fn.expand "%"
+  local buf_id = vim.api.nvim_get_current_buf()
+  for _, pattern in ipairs(options.opts.jupynium_file_pattern) do
+    if utils.string_wildcard_match(filename, pattern) then
```

Comment:
You can instead use `list_wildcard_match` and then remove the for loop?

Reply:
Can you also add a comment that this part is needed when you add lazy loading like `event = "BufWinEnter *.ju.*"` in lazy.nvim?